core/coreunix/add.go

```diff
@@ -470,6 +472,9 @@ func (adder *Adder) addFile(file files.File) error {
 		return err
 	}
 
+	if !strings.EqualFold(adder.WpName, "") && adder.Wrap {
```

Comment:
Do we need an `EqualFold` call to compare to an empty string? I don't think the case is a problem here.